app/syndesis-s2i/pom.xml

```diff
@@ -22,16 +22,16 @@
 
   <parent>
     <groupId>io.syndesis</groupId>
-    <artifactId>syndesis-parent</artifactId>
+    <artifactId>app</artifactId>
```

Comment:
Shouldn't we keep syndesis-parent ? or at least syndesis-app ?

Reply:
syndesis-parent is the parent of app.  So I think we are fine.